sonar-csharp-plugin/pom.xml

```diff
@@ -38,7 +38,7 @@
   </scm>
 
   <properties>
-    <sonarLTS.version>7.9</sonarLTS.version>
+    <sonarLTS.version>9.9</sonarLTS.version>
```

Comment:
Updated to be in sync with the java plugin: https://github.com/SonarSource/sonar-java/pull/4416/files#diff-a2a59812e774224a494679a03de77f5fe24ceb84295e379d6b9583ef97a1ee15R124

Reply:
The variable is not defined on the java plugin, the value is used directly to set the value for <sonarQubeMinVersion> parameter for the `sonar-packaging-maven-plugin` which I think we do use
https://github.com/SonarSource/sonar-dotnet/blob/master/sonar-vbnet-plugin/pom.xml#L187-L195.